adoc/deployment-vmware.adoc

```diff
@@ -40,13 +40,43 @@ for the other method.
 [NOTE]
 ====
 If you choose the {ay} method, please ignore all the following steps for the
-VMWare template creation.
+VMware template creation.
 ====
 
 For each VM deployment, follow the {ay} installation method used for deployment on
 bare metal machines as described in <<deployment_bare_metal>>.
 
-=== Setup using VMWare Template
+=== Setup Using the VMware Template
+
+==== Choose a Disk Format for the Template
+
+Before creating the template, it is important to select the right format of the root hard disk for the node. This format is then used by default when creating new instances with Terraform.
+
+For the majority of cases, we recommend using *Thick Provision Lazy Zeroed*. This format is quick to create, provides good performance, and avoids the risk of running out of disk space due to over-provisioning.
+
+[NOTE]
+====
+It is not possible to resize a disk when using *Thick Provision Eager Zeroed*.
+For this reason, the Terraform variables `master_disk_size` and `worker_disk_size` must be set to the exact same size as in the original template.
```

Comment:
@lcavajani I changed this sentence, because I didn't understand it as it was. Does that still reflect what you wanted to say?